# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/webrtc/signaling/gtest/moz.build
# Commit: 3b777b313b76
# Full Hash: 3b777b313b76b517c66cba5278f06a2da9a6fb55
# Author: Dan Minor <dminor@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 2
# Description:
#   Bug 1654112 - Update VideoConduit to use new APIs. r=ng
#   
#   This also disables the gtests until we figure out the new API.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D130065
# ==============================================================================

diff -r ae0715e32c27 -r 3b777b313b76 media/webrtc/signaling/gtest/moz.build
--- a/media/webrtc/signaling/gtest/moz.build	Fri Oct 30 14:22:28 2020 -0400
+++ b/media/webrtc/signaling/gtest/moz.build	Fri Oct 30 14:30:28 2020 -0400
@@ -45,7 +45,7 @@
         "mediapipeline_unittest.cpp",
         "rtpsources_unittests.cpp",
         "sdp_unittests.cpp",
-        "videoconduit_unittests.cpp",
+#        "videoconduit_unittests.cpp",
     ]
 
     FINAL_LIBRARY = "xul-gtest"
